packages/components/managed-data-table/src/data-table-settings/column-manager/column-manager.spec.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { screen, render } from '../../../../../../test/test-utils';
+import { ColumnManager } from './column-manager';
+
+const createTestProps = (props) => ({
+  availableColumns: [],
+  selectedColumns: [],
+  onUpdateColumns: jest.fn(),
+  ...props,
+});
+
+it('should render hidden columns', () => {
+  const props = createTestProps({
+    availableColumns: [
+      {
+        key: 'column1',
+        label: 'Column 1',
+      },
+    ],
+  });
+
+  render(<ColumnManager {...props} />);
+
+  expect(screen.getByText(/Column 1/i)).toBeInTheDocument();
+});
+
+it('should render visible columns', () => {
+  const props = createTestProps({
+    selectedColumns: [
+      {
+        key: 'column1',
+        label: 'Column 1',
+      },
+    ],
+    availableColumns: [
+      {
+        key: 'column1',
+        label: 'Column 1',
+      },
+    ],
+  });
+
+  render(<ColumnManager {...props} />);
+
+  expect(screen.getByText(/Column 1/i)).toBeInTheDocument();
+});
```

Comment:
I think we should elaborate on this spec a bit. There is more things going on in the column manager which deserve some tests.